test/types/Integral/integralvariable.good

```diff
@@ -0,0 +1 @@
+error: the type of the actual argument 'integral' is generic
```

Comment:
When we get error messages, they are prefixed with the test name and line number of the error occurring.  In this case, instead of "error: ...", it would be "integralvariable.chpl:1: error: ...".  The same goes for the .bad file above